sormas-rest/src/main/resources/workflow-documentation.properties

```diff
@@ -0,0 +1,196 @@
+    Expected response body:<br>\
+    {<br>\
+    &emsp;&emsp;"success" : true,<br>\
+    &emsp;&emsp;"userId" : [some-user-id],<br>\
+    &emsp;&emsp;"token" : [token]<br>\
+    }<br>\
+    The token returned will be used to authenticate in other requests.<br>\
+    <br>\
+  <li><b>Registration of a new person in the external journal</b></li><br>\
+    This process involves several steps that are triggered via the REGISTER button a priviliged user can see in the top right corner when having opened a case or a contact.<br>\
+    To be able to see this button, the user must have at least one of the following user roles: national user, contact supervisor, contact officer, communitiy officer,\
+    surveillance officer, surveillance supervisor, or admin supervisor.<br>\
+    <br>\
+      First comes a SORMAS-internal validation of contact details. The person to be registered needs to have at least an email address or a phone number to pass this validation.\
+      Also, when there are several email addresses or phone numbers, one of them has to be marked primary contact detail, so that it is clear which contact detail \
+      shall be used.<br>\
+      <br>\
+      Then comes an external validation of the contact details. For this, SORMAS fetches an authentication token as in 1. Then it sends a GET request to the following URL for each contact detail to be used in  the external journal:<br>\
+      <tt>GET [interface.patientdiary.probandsurl]/probands?q=[URL-encoded-query-parameter-and-value]'</tt>, with a header like <tt>'x-access-token: [token]'</tt><br>\
+      The [URL-encoded-query-parameter-and-value] consists of a parameter-value-pair. The parameter is either 'Email' or 'Mobile phone'. The value holds the contact detail to be validated.<br>\
+      An unencoded example for this is <tt>"Email" = "example@example.de"</tt><br>\
+      The URL-encoded version is <tt>%22Email%22%20%3D%20%22example%40example.de%22</tt><br>\
+      [token] is replaced with the token fetched for authorization.<br>\
+      The CURL equivalent for an exemplary call is \
+      <tt>curl --request GET 'https://probands-URL.com//probands?q=%22Email%22%20%3D%20%22example%40example.de%22' --header 'x-access-token: my-access-token'</tt><br>\
+      <br>\
+      Expected result is a PatientDiaryQueryResponse which information about any person already registered in the external journal and using the same contact detail.<br>\
+      It needs to be structured as follows:<br>\
+      ``PatientDiaryQueryResponse`` {<br>\
+        &emsp;total&emsp;&emsp;&emsp;integer<br>\
+        &emsp;count&emsp;&emsp;&emsp;integer<br>\
+        &emsp;results&emsp;&emsp;&emsp;List (``PatientDiaryPersonData``)<br>\
+      }<br>\
+      total should state how many persons are registered in the external journal (this information is currently never used in SORMAS).<br>\
```

Comment:
I documented this because I want to get rid of the not used ones. They are coming from climedo.
I want to later create an issue to look over the ones marked as not used. Does that make sense to you?

Reply:
As long as we document this somehow even if it's in a new ticket that's fine by me. Just so we can keep track of this.